core/src/test/java/org/dozer/util/ResourceLoaderTest.java

```diff
@@ -38,13 +39,14 @@ public void testResourceNotFound() throws Exception {
 
   @Test
   public void testGetResourceWithWhitespace() {
-    URL url = loader.getResource(" contextMapping.xml ");
+    URL url = loader.getResource("mappings/contextMapping.xml ");
     assertNotNull("URL should not be null", url);
   }
 
+  @Ignore
```

Comment:
This is a bit of a strange test, as within the code all it does is trim the value. But since the newlines are now mid path (due to the mapping files being moved into resources/mappings/*.xml), then the code doesn't check for this.

I don't see the point in adding code to validate the URL, as this is done by base java code. We can either resolve the URL or we cant.

I would prefer to just remove the test. Does that make sense?

Yes, agree about ignores. All tests should run.